app/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+// strapi service devcontainer
```

Comment:
I think we won't need to have service-specific devcontainer configurations when #22 is merged since we won't mount vscode into a container. I'm just leaving a note so that we remove it in the future.